src/cdk-experimental/menu/menu-item-trigger.ts

```diff
@@ -86,20 +86,28 @@ export class CdkMenuItemTrigger implements OnDestroy {
     return this._overlayRef ? this._overlayRef.hasAttached() : false;
   }
 
+  /**
```

Comment:
Since this doesn't always return an object, an explicit type annotation seems like a good idea here.